### PR TITLE
DIG-979: Move federation config to Vault service store

### PR DESCRIPTION
**Pull request by daisieh**
Using the new version of federation in https://github.com/CanDIG/federation_service/pull/60, we can now initialize our services and servers with the values stored in federation's vault service store. This should still pass integration tests.

### CODE REVIEW

File: lib/federation/initialize.py [hunk 1]
```diff
@@ -37,31 +50,34 @@ def main():
...
 
-    print("Adding servers to federation...")
+    print("Register existing servers")
     url = f"{get_env_value('FEDERATION_PUBLIC_URL')}/v1/servers"
+    response = requests.request("POST", url, headers=headers, params={"register": True})
+    if response.status_code != 200:
+        print(f"POST response: {response.status_code} {response.text}")
+
+    print("Making sure our server is in federation...")
+    server = get_default_server()
```
Comment (OrdiNeu) [inline lib/federation/initialize.py:53]: Best to remove the server = {} json in lines 43-49

File: lib/katsu/katsu_service
Comment (OrdiNeu) [inline lib/katsu/katsu_service:1]: What's the Katsu/OPA changes in this PR?

**Review by OrdiNeu — APPROVED** [review]
Works. Built federation, then cleaned it and edited the setup to remove adding servers/services. Rebuilt it and the old servers/services were still there 👍

File: lib/federation/initialize.py (hunk 1 above)
Comment (daisieh) [reply lib/federation/initialize.py:53]: oh yes, good point.

**Pull request merged**